presto-main/src/main/java/com/facebook/presto/sql/planner/PlanBuilder.java

```diff
@@ -90,6 +90,26 @@ public TranslationMap getTranslations()
         return translations;
     }
 
+    public PlanBuilder prependProjections(Iterable<Expression> expressions, SymbolAllocator symbolAllocator, PlanNodeIdAllocator idAllocator)
```

Comment:
Why do we need this? It shouldn't matter whether the new assignments got first or last in a ProjectNode. All references to its outputs should be symbolic (vs positional) anyway.